frontend/packages/console-plugin-sdk/src/typings/details.ts

```diff
@@ -0,0 +1,23 @@
+import { K8sKind } from '@console/internal/module/k8s';
+import { PageComponentProps } from '@console/internal/components/utils/horizontal-nav';
+import { Extension } from './extension';
+
+namespace ExtensionProperties {
+  export interface DetailViewTab {
+    model: K8sKind;
+    flag?: string;
+    href: string;
+    name: string;
+    // TODO(alecmerdler): Should we allow positioning of the tab...?
+    position?: number;
```

Comment:
Should make it consistent with how we do other extensions like nav items.
But I prefer `weight: number` or using a relative position like `after: string`. Where the `after` value is an id (preferably) or a label.

Reply:
We should just remove it for now, and add it once there is a legitimate use-case.